src/query/interpreter.cpp

```diff
@@ -2216,6 +2223,12 @@ PreparedQuery PrepareCypherQuery(ParsedQuery parsed_query, std::map<std::string,
   if (memory_limit) {
     spdlog::info("Running query with memory limit of {}", utils::GetReadableSize(*memory_limit));
   }
+
+  const auto hops_limit = EvaluateHopsLimit(evaluator, cypher_query->hops_limit_);
+  if (hops_limit) {
+    spdlog::info("Running query with hops limit of {}", *hops_limit);
```

Comment:
I think this should be `spdlog::debug` since we print queries in `spdlog::debug`